src/jsMain/kotlin/io/beatmaps/index/preview.kt

```diff
@@ -36,7 +36,7 @@ class ModalComponent : RComponent<Props, ModalState>() {
     private val modal = createRef<HTMLDivElement>()
     private val iframe = createRef<HTMLIFrameElement>()
 
-    fun show(downloadURL: String) = showIframe("$previewBaseUrl?noProxy=true&url=$downloadURL")
```

Comment:
Do you have functionality in arcviewer for disabling proxying?

Tried to load the previewer locally but it fails to load `https://cors.bsmg.dev/http://localhost:8080/cdn/######.zip`

Reply:
No, I've been disabling proxy based on a whitelist, but I can add it as a URL argument

Reply:
I've just added the functionality to ArcViewer and updated the PR, should work now.